mvc/views/tag-helpers/authoring/sample/TagHlp/src/AuthoringTagHelpers/TagHelpers/zConditionTagHelperCopy.cs

```diff
@@ -0,0 +1,24 @@
+﻿
+using Microsoft.AspNet.Razor.Runtime.TagHelpers;
+
+namespace TagH1p.TagHelpers
+{
+
```

Comment:
What are these cascading impacts you refer to? Why do our samples have to have superfluous whitespace or attributes? These are our samples, they should represent what our customers would see in their own files.